src/nvim/api/private/helpers.c

```diff
@@ -808,6 +811,17 @@ static void init_type_metadata(Dictionary *metadata)
 
   PUT(*metadata, "types", DICTIONARY_OBJ(types));
 }
+static void init_api_level_metadata(Dictionary *metadata)
+{
+  Dictionary version = ARRAY_DICT_INIT;
+  PUT(version, "current", INTEGER_OBJ(NVIM_API_CURRENT));
+  PUT(version, "compatibility", INTEGER_OBJ(NVIM_API_COMPATIBILITY));
+#ifdef NVIM_VERSION_PRERELEASE
+  PUT(version, "prerelease", BOOLEAN_OBJ(true));
```

Comment:
> But then there is no simple way to determine "what is the latest stable version?" 

Well, that's just the API level of the latest Neovim release. I guess it should be mentioned on `--version`? We could document that on the release page if it's too much hassle for developers to have a Neovim release version installed and check that.

> My point is simply that both plugins that wants to only use stable and those that wants to try use unstable api level (if available) should be allowed to do so based on the global fields without deep inspection.

My idea was that you can don't need the prerelease field to get this "global information", you can simply check current API level > latest stable API level. I understood that we bump the API level on `master` as soon as an API change is introduced.